### PR TITLE
chore: update commands to match other Svelte projects

### PR DESCRIPTION
**Pull request by benmccann**

### CODE REVIEW

**Review by manuel3108 — DISMISSED** [review]
I additionally updated the docs

File: package.json [hunk 1]
```diff
@@ -9,8 +9,8 @@
         "prettier:fix": "pnpm prettier . --write",
```
Comment (AdrianGonz97) [inline package.json:9]: while we're at it, perhaps we can also update format and lints as well?
```suggestion
        "format": "prettier . --write",
```

Comment (benmccann) [reply package.json:9]: good idea!

Comment (manuel3108) [reply package.json]: Sounds smart. How would you proceed with the other commands?

```bash
"prettier:check": "pnpm prettier . --check",  # format:check
"prettier:fix": "pnpm prettier . --write", # format
"eslint:check": "pnpm eslint .", # lint
"eslint:fix": "pnpm eslint --fix .", # lint:check
```

File: .github/workflows/ci.yml [hunk 2]
```diff
@@ -56,7 +56,7 @@ jobs:
               run: pnpm types:check
```
Comment (AdrianGonz97) [inline .github/workflows/ci.yml:56]: we can probably rename this one to just `check` as well and we should be good to go

File: package.json [hunk 3]
```diff
@@ -5,12 +5,10 @@
...
     "scripts": {
         "ci:publish": "changeset publish",
-        "prettier:check": "pnpm prettier . --check",
-        "prettier:fix": "pnpm prettier . --write",
-        "eslint:check": "pnpm eslint .",
-        "eslint:fix": "pnpm eslint --fix .",
-        "build:dev": "rollup --config --watch",
-        "build:prod": "rollup -c",
+        "format": "prettier . --write && pnpm eslint --fix .",
+        "lint": "pnpm prettier . --check && pnpm eslint .",
```
Comment (AdrianGonz97) [inline package.json:9]:
```suggestion
        "format": "prettier . --write && eslint --fix .",
        "lint": "prettier . --check && eslint .",
```

File: .github/workflows/ci.yml (hunk 2 above)
Comment (benmccann) [reply .github/workflows/ci.yml]: done

**Review by AdrianGonz97 — APPROVED** [review]
awesome!!

**Review by manuel3108 — APPROVED** [review]

**Pull request merged**